fn.go

```diff
@@ -40,6 +42,15 @@ type Voter struct {
 	Status string `json:"status"`
 }
 
+type initialResources struct {
+	Input            *v1beta1.Input
+	API              *slack.Client
+	XR               *resource.Composite
+	Rsp              *fnv1beta1.RunFunctionResponse
+	CurrentTimestamp int
+	Desired          map[resource.Name]*resource.DesiredComposed
+}
+
```

Comment:
@ca7alindev not sure is good idea to collect this resources under structure. If you need variables just make variables, structures is used if you have multiple objects of this structure, but looks like not, or i oversaw something?